torch/_inductor/triton_heuristics.py

```diff
@@ -280,17 +281,20 @@ def run(self, *args, grid, stream):
         super().run(*args, grid=grid, stream=stream)
```

Comment:
Do we still need run the kernel even if we don't want to profile it?

Reply:
I think that's technically true, but I think this is more convenient to ensure that our code matches up with w.e. eager is doing.

Reply:
hmm, but in eager we do need run every kernel?

If we skip running the kernel because it's not selected by the profile regex, the whole model may produce incorrect results.

Reply:
It's not an issue unless we have indirect indexing, since Inductor always already has the out tensors allocated before the kernel runs.

Reply:
yep, it wouldn't crash. I'm mainly talking about numerical. But maybe we don't care about numerical when we are doing profiling :)